src/context/librariesContext.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createContext, Dispatch, useEffect, useState } from 'react';
+import { LibraryData } from '../components/RegistryAdmin';
+
+export type LibrariesContextValues = {
+  librariesInContext: LibraryData[];
+  setLibrariesInContext: Dispatch<React.SetStateAction<LibraryData[]>>;
```

Comment:
yeah let's just make these `libraries` and `setLibraries`. It's clear through the rest of the context where they're coming from.